wikidata/globecoordinate.py

```diff
@@ -0,0 +1,42 @@
+from .entity import Entity
```

Comment:
For Sphinx rendering:

```suggestion
""":mod:`wikidata.globecoordinate` --- Globe coordinate
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

"""
from .entity import Entity
```